src/cephadm/cephadm

```diff
@@ -1978,6 +2011,7 @@ def command_inspect_image():
 
 ##################################
 
+@default_image
```

Comment:
@ricardoasmarques you're ok with this change?

Reply:
`ceph-salt` will always explicitly set the image to use when calling `cephadm bootstrap`, so lgtm.